### ISSUE TITLE
libWrapper error in Compendium Folders

### ISSUE THREAD

**Issue by iotech-fvtt** [earlSt1/vtt-compendium-folders#138]
**Self hosted. Mods enabled:**
- libWrapper 1.12.4.0
- Compendium Folders 2.4.3
- Sidebar Macros 0.8.0

**_Description_**: Popup red toast. Error actually caused by Sidebar Macros but libWrapper is blaming it on Compendium Folders. Doesnt seem to prevent anything from actually working, but libWrapper does the popup on every refresh.

_**Console dump:**_

```
compendium-folders.js:1890 Uncaught (in promise) LibWrapperPackageError: Error detected in module Compendium Folders.

This is *not* a libWrapper bug.

Find information about this module here: https://github.com/earlSt1/vtt-compendium-folders

Ask the community for support:
- FoundryVTT Discord's #modules-troubleshooting channel: https://discord.gg/foundryvtt
- FoundryVTT Reddit: https://www.reddit.com/r/FoundryVTT

== Technical Details:
Detected by libWrapper.
Package ID= compendium-folders
Error= Could not find target 'MacroSidebarDirectory.prototype._getFolderContextOptions': Could not find root scope 'MacroSidebarDirectory'.

[Detected 1 package: compendium-folders]
    at ae (libWrapper-api.js:135:10)
    at se (libWrapper-api.js:152:19)
    at Function.🎁register [as register] (libWrapper-api.js:522:14)
    at Function.setup (fic-folders.js:575:24)
    at compendium-folders.js:1869:24
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at Game.initialize (foundry.js:5421:11)
    at Wrapper.🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:511:22)
    at Game.🎁libWrapperInit (libWrapper-api.js:793:11)
```

**Comment by earlSt1** [comment]
Thanks for the bug report, looks like Sidebar Macros has had an update recently which has broken the integration, will have a look soon

**Comment by arcanistzed** [comment]
I switched to using `esmodules` in my manifest which means that `MacroSidebarDirectory` could no longer be found in global scope. This should be fixed in v0.8.1 of Sidebar Macros arcanistzed/sidebar-macros@8e484a8d4410e4d6bf405fa01474f149fcb5f20f.
